src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,114 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int AGE_BARELY_OK = 18;
+    private static final int AGE_OK = 25;
+    private static final int AGE_NEGATIVE = -10;
+    private static final int AGE_NOT_OK = 13;
+    private static final String SHORT_PASSWORD = "123";
+    private static final String BLANK_LOGIN = "";
+    private static final String LOGIN_WITH_SPACES = "123123  ";
+    private static StorageDao storageDao;
+    private static User validUser1;
+    private static User validUser2;
+    private static User validUser3;
+    private static User validUser4;
+    private static RegistrationServiceImpl registrationServiceImpl;
+
+    @BeforeAll
+    static void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationServiceImpl = new RegistrationServiceImpl();
+        validUser1 = new User("login1", "1234567", AGE_BARELY_OK);
+        validUser2 = new User("login2", "12345678", 26);
+        validUser3 = new User("login3", "123456789", 27);
+        validUser4 = new User("login4", "1234567890", 28);
+    }
+
+    @BeforeEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void registerValidUser_Ok() {
```

Comment:
```suggestion
    void register_validUser_ok() {
```
likewise in the following cases. And don't forget access modifiers